components/camel-elasticsearch5-rest/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>components</artifactId>
+    <version>2.20.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.apache.camel</groupId>
+  <artifactId>camel-elasticsearch5-rest</artifactId>
+  <packaging>jar</packaging>
+  <name>Camel :: ElasticSearch5 :: REST</name>
+  <description>Camel ElasticSearch 5.x REST support</description>
+
+  <properties>
+    <elasticsearch.version>${elasticsearch5-version}</elasticsearch.version>
+    <camel.osgi.export.pkg>org.apache.camel.component.elasticsearch5.*;${camel.osgi.version}</camel.osgi.export.pkg>
+    <camel.osgi.export.service>org.apache.camel.spi.ComponentResolver;component=elasticsearch5</camel.osgi.export.service>
```

Comment:
The name should not clash with the other, eg should be `elasticsearch5-rest` or `elasticsearch-rest`